src/components/SingleArticle.jsx

```diff
@@ -28,9 +34,26 @@ const SingleArticle = () => {
                 <Card.Text>{article.body}</Card.Text>
             </Card.Body>
         </Card>
-
+            <div>
+            <h2 className="commentsContainer">Comments</h2>
+            {comments.length === 0 ? (
+                <p>No comments</p>
```

Comment:
Well done getting this in there - if I go to articles/37 I get a nice message telling me the above!